packages/web-components/src/components/va-alert-expandable/va-alert-expandable.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  Listen,
+  State,
+  h,
+} from '@stencil/core';
+import classnames from 'classnames';
+
+@Component({
+  tag: 'va-alert-expandable',
+  styleUrl: 'va-alert-expandable.css',
+  shadow: true,
+})
+export class VaAlertExpandable {
+  @Element() el: HTMLElement;
+
+  @State() open: boolean;
+
+  /**
+   * Determines the icon and background color.
+   * One of `info`, `error`, `success`, `warning`, or `continue`
+   */
+   @Prop() status?: 'continue' | 'error' | 'info' | 'success' | 'warning'; 
+
+  /**
+   * The text to trigger the expansion
+   */
+  @Prop() trigger: string = '';
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics: boolean = false;
+
+  /**
+   * If true, the status icon is removed.
+   */
+  @Prop() iconless: boolean = false;
+
+  /**
+   * The event used to track usage of the component. This is emitted when an
+   * anchor link is clicked and disableAnalytics is not true.
+   */
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  @Listen('resize', { target: 'window' })
+  handleResize() {
+    this.updateAlertBodyMaxHeight();
+  }
+
+  toggleOpen(): void {
+    // Conditionally track the event.
+    if (!this.disableAnalytics) {
+      this.componentLibraryAnalytics.emit({
+        componentName: 'va-alert-expandable',
+        action: !this.open ? 'expand' : 'collapse',
+        details: {
+          triggerText: this.trigger,
+        },
+      });
```

Comment:
I believe we need to reach out to Brian to track this and add it on vets-website [component-library-analytics-setup](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/component-library-analytics-setup.js).

Reply:
@andresriveratoro , is that something you all do as part of onboarding a new component, or is it something that I need to reach out and do prior to?

Reply:
@k80bowman @bsmartin-ep what is the way that you would like to have this event added to GTM? Should we do this as a separate ticket?

Reply:
If we end up leaving this work to the contributor, we should update our README or include this information somewhere more easily accessible than our Confluence.

Reply:
@longmd @andresriveratoro can help with adding analytics. Everything else should be good to go here. Thank you for working on this!

Reply:
@andresriveratoro , does Brian need to do the setup on their end before I add to the events setup object in `vets-website`?

Reply:
@longmd Brian has to setup the analytics in Google as well for it to work properly, but you're good to go ahead and merge this PR. I'll create a ticket to update vets-website and assign it to you. Thanks!